addons/mail/static/src/models/channel_member_view.js

```diff
@@ -15,17 +15,17 @@ registerModel({
          * @param {MouseEvent} ev
          */
         onClickMember(ev) {
-            if (isEventHandled(ev, 'PersonaImStatusIcon.Click') || !this.channelMember.partner) {
+            if (isEventHandled(ev, 'PersonaImStatusIcon.Click') || !this.channelMember.persona.partner) {
                 return;
             }
-            this.channelMember.partner.openChat();
+            this.channelMember.persona.partner.openChat();
         },
         /**
          * @private
          * @returns {Boolean}
          */
         _computeHasOpenChat() {
-            return this.channelMember.partner ? true : false;
+            return this.channelMember.persona.partner ? true : false;
```

Comment:
Note that MessageView has also a compute of this kind (`MessageView/hasAuthorOpenChat`). Both seem to be a bit redundant; perhaps the two fields could be merged in the future?